python/py-lockfile/Portfile

```diff
@@ -21,11 +22,18 @@ long_description  \
     also provided, more as a demonstration of the possibilities it provides \
     than as production-quality code.
 
-homepage            https://code.google.com/p/${real_name}/
-master_sites        googlecode:${real_name} http://distfiles.macports.org/python/
-distname            lockfile-${version}
-checksums           md5     4e4c7ea4c4301500da5e7f3b51e01cfe \
-                    sha1    1eebaee375641c9f29aeb21768f917dd2b985752 \
-                    rmd160  c94c1a638f2a11181063d6025a6c5acc8c9d09e6
+homepage            https://pypi.org/project/lockfile/
+
+checksums           rmd160  6aaa40fb1a1bd4322f26182fca631a6c4c1f8d24 \
+                    sha256  6aed02de03cba24efabcd600b30540140634fc06cfa603822d508d5361e9f799 \
+                    size    20874
+
+python.versions     27 37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append port:py${python.version}-setuptools \
+                         port:py${python.version}-pbr
+
+    livecheck.type       none
```

Comment:
the three lines here in the ```if-block``` do not conform to the modeline; please adjust the whitespace so that the statements are aligned with multiples of four spaces